arviz/stats/stats.py

```diff
@@ -352,16 +389,24 @@ def loo(data, pointwise=False, reff=None):
     reff : float, optional
         Relative MCMC efficiency, `effective_n / n` i.e. number of effective samples divided by
         the number of actual samples. Computed from trace by default.
+    scale : str
+        Output scale for loo. Available options are:
+
+        - `deviance` : (default) -2 * (log-score)
+        - `log` : 1 * log-score (after Vehtari et al. (2017))
+        - `negative_log` : -1 * (log-score)
 
     Returns
     -------
-    DataFrame with the following columns:
+    pandas.Series with the following columns:
```

Comment:
Perhaps columns should be changed to index here because Series don't technically have columns